nuxeo-features/nuxeo-automation/nuxeo-automation-server/src/main/resources/skin/views/doc/style.css

```diff
@@ -142,34 +142,43 @@ pre {
   white-space: pre;
 }
 
-header[role="banner"] {
-  background-color: #1b5983;
-  color: #fff;
+header[role="banner"]{
+  background-color: #FFFFFF;
+  color: #0066FF;
   font-family: sans-serif;
   vertical-align: middle;
+  box-shadow: 0 1px 3px rgba(0,0,0,0.4);
```

Comment:
I suggest to respect the alphabetical order of the css properties.
(So here: background-color, box-shadow, color..)
Same for your other css changes.

Reply:
Otherwise 👍